### PR TITLE
Redesign App Structure

### PR DESCRIPTION
**Pull request by akshay2905**
First set of commit for the restructuring of the App to a more generic design which would enable reusability.

### CODE REVIEW

File: src/components/views/reportview/ReportView.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+class ReportView extends React.Component {
+
+render(){
+
+	return(
+	<div className = "col-lg-10" >
+	    <div id = "vis-container" >
+	        <div id="card-container">
+	          Hello World{/*
```
Comment (gggodhwani) [inline src/components/views/reportview/ReportView.js]: Can you check this, please?

File: src/components/views/reportview/ReportView.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+	            {
+	                this.state.viewBy == "choropleth" ? ( 
+	                    <Choropleth data={this.state.indicatorData} attrType={this.state.budgetAttr} selectedIndicator={this.state.indicatorData.indicator} selectedSector = {this.state.sectorSelected} figureUnit = {this.state.indicatorData.unit} /> ) 
+	                :(
+	                    <GraphComponent data={this.state.indicatorData} attrType={this.state.budgetAttr} selectedIndicator={this.state.indicatorData.indicator} selectedSector = {this.state.sectorSelected} sectorName= {this.state.sectorName} /> )
+	            }
+
+	        <div className="row indicator-description">
+	          Source - {this.state.notesText.source}  
+	        </div>
```
Comment (gggodhwani) [inline src/components/views/reportview/ReportView.js]: Do we need to keep these comments? If yes, add a rationale above it with TODO tag.

Comment (akshay2905) [inline src/components/views/reportview/ReportView.js]: This is just kept in place to complete the design at the structural level. Would be developing it further. Current that component isnt being used in the app. Once developed fully, we would include it.

Comment (gggodhwani) [inline src/components/views/reportview/ReportView.js]: Ok add a TODO tag mentioning the same

**Pull request merged**